web/src/test/installers.ts

```diff
@@ -492,4 +521,32 @@ spec:
       expect(i.flags()).to.equal(`velero-namespace=velero`);
     });
   });
+        
+  describe("fluentd minimum spec flags", () => {
+    it("should not generate any flags", () => {
+
+        const i = Installer.parse(fluentdMin);
```

Comment:
indentation

Reply:
this is a dupe test actually ill delete it